schema_based_on_diagram.sql

```diff
@@ -0,0 +1,49 @@
+//Database schema based on a given diagram
+
+CREATE TABLE patients (
+    id serial primary key NOT NULL,
+    name varchar,
+    date_of_birth date
+);
+
+CREATE TABLE medical_histories (
+    id serial primary key NOT NULL,
+    admitted_at timestamp,
+    patient_id int,
+    status varchar ,
+    PRIMARY KEY(id),
+    CONSTRAINT fk_patient,
+      FOREIGN KEY(id) ,
+	  REFERENCES patient(patient_id)
+);
+
+CREATE TABLE treatments (
+    id serial primary key NOT NULL,
+    type varchar,
+    name varchar 
+    CONSTRAINT fk_patient
+    FOREIGN KEY(id) REFERENCES medical_histories(id)
+);
```

Comment:
- [ ] As it is written in the project requirements, you are also required to create all join tables wherever necessary. Kindly ensure that you create the join table. **Hint:** a table between medical_histories and treatments

![image](https://user-images.githubusercontent.com/86318284/207371088-508efa7b-f32a-4051-9de1-a4c38e16ddf2.png)